scripts/plot_dataframe.py

```diff
@@ -74,52 +76,70 @@ def load_trades(args: Namespace, pair: str, timerange: TimeRange) -> pd.DataFram
         # must align with columns in backtest.py
         columns = ["pair", "profit", "opents", "closets", "index", "duration",
                    "open_rate", "close_rate", "open_at_end", "sell_reason"]
-        with file.open() as f:
-            data = json.load(f)
-            trades = pd.DataFrame(data, columns=columns)
-        trades = trades.loc[trades["pair"] == pair]
-        if timerange:
-            if timerange.starttype == 'date':
-                trades = trades.loc[trades["opents"] >= timerange.startts]
-            if timerange.stoptype == 'date':
-                trades = trades.loc[trades["opents"] <= timerange.stopts]
-
-        trades['opents'] = pd.to_datetime(trades['opents'],
-                                          unit='s',
-                                          utc=True,
-                                          infer_datetime_format=True)
-        trades['closets'] = pd.to_datetime(trades['closets'],
-                                           unit='s',
-                                           utc=True,
-                                           infer_datetime_format=True)
+        if os.path.exists(file):
```

Comment:
since file is a `Path` object, you can do `if file.exists()`. 

[docs](https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module)